pkg/csi/service/wcpguest/controller.go

```diff
@@ -420,6 +456,137 @@ func (c *controller) ControllerPublishVolume(ctx context.Context, req *csi.Contr
 	return resp, nil
 }
 
+// controllerPublishForFileVolume is a helper mthod for handling ControllerPublishVolume request for File volumes
+func controllerPublishForFileVolume(ctx context.Context, req *csi.ControllerPublishVolumeRequest, c *controller) (
```

Comment:
This could be a method under `controller` struct.

Reply:
Keeping the methods as util/helper methods within the `wcpguest` package for now